docs/network/interfaces_and_networks.md

```diff
@@ -47,40 +47,47 @@ fields:
 </tr>
 <tr class="even">
 <td><p><code>multus</code></p></td>
-<td><p>Secondary network provided using Multus</p></td>
```

Comment:
Please consider replacing the HTML code with markdown [tables](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables).
It will improve the document's consistency and the end result is much shorter.

Reply:
Good suggestion! done.